examples/ex9p.cpp

```diff
@@ -407,7 +437,7 @@ int main(int argc, char *argv[])
    // 10. Define the time-dependent evolution operator describing the ODE
    //     right-hand side, and perform time-integration (looping over the time
    //     iterations, ti, with a time-step dt).
-   FE_Evolution adv(*M, *K, *B, *fes);
+   FE_Evolution adv(*M, *K, *B, *fes, pa);
```

Comment:
To make the code more similar to the serial version, it seems more natural to pass the `ParBilinearForm`s as parameters here.

Reply:
Do you mean both `m` and `k`? That's not very elegant because it's redundant I think.

Reply:
I'll try to initialize `M` and `K` inside the `FE_Evolution`

Reply:
I changed the interface of `FE_Evolution` to be similar to `ex9`, I'm not completely convinced about the elegance of the code, tell me if you like it.

Reply:
I think you can use `OperatorPtr` in the `FE_Evolution` class to simplify the code a little but I'll leave that up to you.